lib/cpp/src/thrift/transport/TSSLSocket.cpp

```diff
@@ -249,6 +249,17 @@ TSSLSocket::~TSSLSocket() {
   close();
 }
 
+bool TSSLSocket::hasPendingDataToRead() {
+  if (!isOpen()) {
+    return false;
+  }
+  initializeHandshake();
+  if (!checkHandshake())
+    throw TSSLException("SSL_peek: Handshake is not completed");
```

Comment:
This should probably say something other than SSL_peek?

Reply:
Fixed.